.github/actions/openvino_provider/action.yml

```diff
@@ -0,0 +1,157 @@
+name: 'OpenVINO provider'
+description: 'Provides OpenVINO build artifacts from a requested source'
+inputs:
+  revision:
+    description: 'Version of OpenVINO to provide - latest_available | HEAD | specific commit hash'
+    required: true
+  branch_name:
+    description: 'Branch of OpenVINO to take the revision from if no specific hash was provided. 
+                  Taken from github context by default'
+    required: false
+  ov_artifact_name:
+    description: "Name under which to upload provided OpenVINO build artifacts, set automatically by default"
+    required: false
+  commit_packages_to_provide:
+    description: "Comma-separated list of OpenVINO packages from post-commit to provide ('all' to get everything)"
+    required: false
+  commit_build_key:
```

Comment:
What is it for?

Reply:
To choose the artifacts for the correct platform (os + arch + release/debug + potentially some other build variations, identified by a single key)